### ISSUE TITLE
Missing documentation for onset, duration, description attributes in Annotations class?

### ISSUE THREAD

**Issue by scott-huberty** [mne-tools/mne-python#12379]
### Proposed documentation enhancement

A colleague of mine who is a bit of an MNE power user had no idea that you can do something like `raw.annotations.onset` to retrieve an array of the onsets of all individual annotations:

```
import mne

annot = mne.Annotations(onset=[1, 3, 5], duration=[1, 1, 1], description=['a', 'b', 'c'])
annot.onset 
# array([1., 3., 5.])
```

 As far as I can tell we don't document the `onset`, `duration`, `description` attributes in the [API documentation](https://mne.tools/dev/generated/mne.Annotations.html). Should we?

**Comment by larsoner** [comment]
Yeah we could add them. I think we get them for free (maybe?) if we do:
```
def __init__(self, ...)
    ...
    self._onset, self._duration, ... = ...

@property
def onset(self):
    """..."""
    return self._onset

@onset.setter
def onset(self, onset)
    self._onset[:] = onset
```
Or something like this

**Comment by scott-huberty** [comment]
Ok I'll test it out and open a PR when I have some time.

**Comment by hoechenberger** [comment]
(Still thinking these should all be pluralized but this is a different discussion I suppose)

**Comment by Famous077** [comment]
Hi @scott-huberty , I’d like to work on this documentation improvement. I can add clear API docs for the `onset`, `duration`, and `description` attributes of `Annotations`, along with a short example showing how to access them. Could you please assign this issue to me?

**Comment by tsbinns** [comment]
@Famous077 Thanks for wanting to contribute. We don't typically assign issues to users. Just feel free to open a PR whenever you have something.

**Comment by Famous077** [comment]
@tsbinns Thanks for the clarification. I've already implemented the fix and 
raised a PR. Happy to update based on your feedback.
